### ISSUE TITLE
mariadb-server 10.0.26 /var/mysql doesn't exist

### ISSUE THREAD

**Issue by churnd** [joyent/smartos-live#735]
Getting this error when I try to install mariadb-server 10.0.26:

[ Jul 23 18:20:16 Leaving maintenance because clear requested. ]
[ Jul 23 18:20:16 Enabled. ]
[ Jul 23 18:20:16 Executing start method ("/opt/local/lib/svc/method/mariadb start"). ]
[ Jul 23 18:20:16 svc.startd could not set context for method:  ]
[ Jul 23 18:20:16 chdir: No such file or directory ("/var/mysql") ]
[ Jul 23 18:20:16 Method "start" exited with status 96. ]
[ Jul 23 18:20:32 Leaving maintenance because disable requested. ]
[ Jul 23 18:20:32 Disabled. ]

**Comment by rmustacc** [comment]
@churnd can you share what pkgsrc quarter / image this is based on?
@jperkin, @mamash can one of you look at this?

**Comment by churnd** [comment]
What information exactly do you need?

```[root@smartos /opt/json]# imgadm list
UUID                                  NAME          VERSION   OS       TYPE          PUB
7b5981c4-1889-11e7-b4c5-3f3bdfc9b88b  ubuntu-16.04  20170403  linux    lx-dataset    2017-04-03
f6acf198-2037-11e7-8863-8fdd4ce58b6a  base-64       17.1.0    smartos  zone-dataset  2017-04-13
```

```
[root@zone ~]# pkgin -v     
pkgin 0.9.4 for SunOS-5.11 x86_64 (using SQLite 3.17.0)
```

**Comment by mamash** [comment]
This issue is now fixed and available as part of the latest mariadb-10.0.32nb1 packages in 2016Q4 LTS. The 2017Q1 release normally doesn't get updates, but we can fix this there too, if you need it.

**Comment by mamash** [comment]
Fixed in 2017Q2 too, we suggest you pick either the 2016Q4 (LTS) or the 2017Q2 (current) release for the fix.
